src/apps/AppCenter/Screenshots.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { theme } from '@aragon/ui'
+import { GU } from '../../utils'
+
+class Screenshots extends React.Component {
+  static propTypes = {
+    screenshots: PropTypes.arrayOf(PropTypes.string),
+  }
+  state = {
+    currentScreenshot: -1,
+  }
+  open(index) {
+    this.setState({ currentScreenshot: index })
```

Comment:
Note: we should probably implement this 😄

Reply:
Ah yes, we decided to remove this feature and redesign the screen differently. I’ll remove this.